### PR TITLE
Prepare guest counters for more than 32767 guests

### PR DESCRIPTION
**Pull request by Gymnasiast**

### CODE REVIEW

File: src/openrct2-ui/windows/GuestList.cpp [hunk 1]
```diff
@@ -152,13 +152,13 @@ static uint32_t _window_guest_list_last_find_groups_tick;
...
-static uint32_t _window_guest_list_selected_view;    // 0x00F1EE13
-static uint16_t _window_guest_list_num_pages;        // 0x00F1EE08
-static int32_t _window_guest_list_num_groups;        // 0x00F1AF22
+static uint32_t _window_guest_list_highlighted_index; // 0x00F1EE10
+static int32_t _window_guest_list_selected_tab;       // 0x00F1EE12
+static int32_t _window_guest_list_selected_filter;    // 0x00F1EE06
+static int32_t _window_guest_list_selected_page;      // 0x00F1EE07
+static uint32_t _window_guest_list_selected_view;     // 0x00F1EE13
+static uint16_t _window_guest_list_num_pages;         // 0x00F1EE08
+static uint32_t _window_guest_list_num_groups;        // 0x00F1AF22
```
Comment (tupaschoal) [inline src/openrct2-ui/windows/GuestList.cpp]: Are these comments still relevant?

File: data/language/en-GB.txt [hunk 2]
```diff
@@ -1188,7 +1188,7 @@ STR_1813    :Miscellaneous Objects
 STR_1814    :Actions
 STR_1815    :Thoughts
 STR_1816    :{SMALLFONT}{BLACK}Select information type to show in guest list
-STR_1817    :({COMMA16})
+STR_1817    :({COMMA32})
```
Comment (tupaschoal) [inline data/language/en-GB.txt]: Should we pursue these changes in `Localisation` ourselves? Seems unnecessary to involve translators if there is nothing to translate

Comment (Gymnasiast) [reply data/language/en-GB.txt]: Sure. Do note that we still have to change it in the `Localisation` repository and not here.

File: src/openrct2-ui/windows/GuestList.cpp (hunk 1 above)
Comment (Gymnasiast) [reply src/openrct2-ui/windows/GuestList.cpp]: Personally, I prefer keeping them. But I'm not too fussed either way.

Comment (tupaschoal) [reply src/openrct2-ui/windows/GuestList.cpp]: Alright, then let's keep'em.

File: data/language/en-GB.txt (hunk 2 above)
Comment (tupaschoal) [reply data/language/en-GB.txt]: Yes yes, I was just raising the point

**Comment by duncanspumpkin** [comment]
@Gymnasiast I'm going to rework the guestlist so it doesn't need to use `var_492` in #11014. So we could remove that from this PR.

**Comment by Gymnasiast** [comment]
@duncanspumpkin I'd prefer if this one was merged as-is if rebasing yours isn't too much effort. Otherwise I'll look into that.

**Comment by duncanspumpkin** [comment]
Okay fine by me

**Comment by duncanspumpkin** [comment]
Argh sorry completely forgot i was going to wait for your refactor first.

**Comment by Gymnasiast** [comment]
Ah, well, no harm done. Could you approve/request changes?

File: src/openrct2/interface/Window_internal.h [hunk 3]
```diff
@@ -71,6 +71,7 @@ struct rct_window
     {
         int16_t picked_peep_frame; // Animation frame of picked peep in staff window and guest window
         int16_t var_492;
+        uint32_t NumGuests;
```
Comment (duncanspumpkin) [inline src/openrct2/interface/Window_internal.h]: I can't see this being used anymore after my change.

Comment (Gymnasiast) [reply src/openrct2/interface/Window_internal.h]: You're right. I have removed it.

**Pull request merged**